TerminalDocs/tutorials/ssh.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Windows Terminal SSH
+description: In this tutorial, learn how to set up an SSH connection in the Windows Terminal.
+author: cinnamon-msft
+ms.author: cinnamon
+ms.date: 05/19/2020
+ms.topic: tutorial
+ms.service: terminal
+#Customer intent: As a developer or IT admin, I want to set up am SSH connection in Windows Terminal so that I can connect to other servers.
+---
+
+# Tutorial: SSH in Windows Terminal
+
+Windows 10 has a build-in SSH client that you can use in the Windows Terminal.
+
+In this tutorial, you'll learn how to set up a profile in Windows Terminal that uses SSH.
+
+## Create a profile
```

Comment:
Is there any good documentation on SSH we can link below in a new "Resources" section?